### PR TITLE
Release 0.108.0

### PR DESCRIPTION
**Pull request by kitsonk**

### CODE REVIEW

**Comment by bartlomieju** [comment]
Is the failure on `canary` a spurious event or something's actually wrong?

**Comment by bartlomieju** [comment]
> Is the failure on `canary` a spurious event or something's actually wrong?

Looks like it is not spurious. We probably need to change https://github.com/denoland/dotland/pull/1888 to `0.14.1`, `0.107.0` pair. Reset CI on this PR half and hour later, and then add another PR to https://github.com/denoland/dotland once this PR is released.

**Review by ry — APPROVED** [review]

File: node/_fs/_fs_readFile_test.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ Deno.test("readFileEncodeBase64Success", async function () {
     });
   });
   assertEquals(typeof data, "string");
-  assertEquals(data as string, "aGVsbG8gd29ybGQ=");
+  assertEquals(data as string, "aGVsbG8gd29ybGQA");
```
Comment (nayeemrmn) [inline node/_fs/_fs_readFile_test.ts:61]: This is incorrect, Node returns the previous value. I'm not sure what changed between CLI versions though, looks like something to do with `Uint8Array::subarray()`.

Comment (kitsonk) [reply node/_fs/_fs_readFile_test.ts:61]: decode `aGVsbG8gd29ybGQA` results in `hello world` as well... this has been failing in canary for quite a while without anyone addressing it... 😢

Comment (nayeemrmn) [reply node/_fs/_fs_readFile_test.ts:61]: @kitsonk This is the fix: https://github.com/denoland/deno_std/pull/1279/commits/c0d7da82c48cceddf16cd631818175e298f957ab, the fixture changes there align with Node. It was an std issue probably exposed by some V8 patch.

Comment (kitsonk) [reply node/_fs/_fs_readFile_test.ts:61]: I am incorporating into this PR.

**Pull request merged**